### PR TITLE
Do not expose port out of ActivityIframeView

### PR DESCRIPTION
**Pull request by sohanirao**
Activity Port doesn't need to be exposed from ActivityIframeView. It is already passed into the constructor. Handle the only case where the port is used from this Object by providing another API. This change enables ActivityIframeView to potentially use another ActivityPort internally that can communicate differently (use Protos) than existing ActivityPort.

### CODE REVIEW

File: src/ui/activity-iframe-view.js [hunk 1]
```diff
@@ -124,28 +125,29 @@ export class ActivityIframeView extends View {
    * @return {!Promise}
    */
   onOpenIframeResponse_(port, dialog) {
-    this.port_ = port;
+    this.portResolved_ = port;
```
Comment (chenshay) [inline src/ui/activity-iframe-view.js]: I'd rather call this port_ and the method portPromise_()

File: src/ui/activity-iframe-view.js [hunk 2]
```diff
@@ -124,28 +125,29 @@ export class ActivityIframeView extends View {
...
-    return this.port_.whenReady();
+    return this.portResolved_.whenReady();
   }
 
   /**
    * @return {!Promise<!web-activities/activity-ports.ActivityIframePort>}
+   * @private
    */
-  port() {
+  port_() {
```
Comment (chenshay) [inline src/ui/activity-iframe-view.js]: change to portPromise_

File: src/ui/activity-iframe-view.js (hunk 1 above)
Comment (sohanirao) [reply src/ui/activity-iframe-view.js]: there's already a variable called portPromise_ on Line 91

File: src/ui/activity-iframe-view.js (hunk 2 above)
Comment (chenshay) [reply src/ui/activity-iframe-view.js]: then change to getPortPromise_

Comment (sohanirao) [reply src/ui/activity-iframe-view.js]: done

**Pull request merged**